MVC/model.py

```diff
@@ -183,9 +183,9 @@ def reset(self):
     def save(self, filename):
         if self.agent:
             self.agent.save(filename)
-            if (self.cloudBridge is not None):
-                if os.path.exists(filename):
-                    self.cloudBridge.upload(filename)
+            #if (self.cloudBridge is not None):
+            #    if os.path.exists(filename):
+            #        self.cloudBridge.upload(filename)
```

Comment:
Why is this segment being commented out and not just deleted?

Reply:
It could be deleted